pkg/minikube/cluster/cluster.go

```diff
@@ -62,6 +64,11 @@ const (
 //INFO lvl logging is displayed due to the kubernetes api calling flag.Set("logtostderr", "true") in its init()
 //see: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/logs/logs.go#L32-L34
 func init() {
+	provision.Register("minishift", &provision.RegisteredProvisioner{
```

Comment:
>  As said, let's avoit init(). Put it into start.go for example.

Ah right, there was existing init() method on cluster.go so I hooked it there but yes putting it to start.go make much more sense.